src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -144,6 +144,10 @@ export interface GetConceptsResponse {
   result: Concept[];
 }
 
+export interface SearchConceptsResponse {
+  result: Concept[];
+}
```

Comment:
so the style protocol is not to reuse the same object (GetConceptsResponse), but make a new one for this purpose. Makes sense, it's clear. Just wondering, does that same expectation extend to API return types?